src/Azure.Functions.Cli/Kubernetes/KEDA/V1/KedaV1ResourceFactory.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Functions.Cli.Kubernetes.KEDA.V1.Models;
+using Azure.Functions.Cli.Kubernetes.Models;
+using Azure.Functions.Cli.Kubernetes.Models.Kubernetes;
+using Newtonsoft.Json.Linq;
+
+namespace Azure.Functions.Cli.Kubernetes.KEDA.V1
+{
+    public class KedaV1ResourceFactory
+    {
+        internal static ScaledObjectV1Alpha1 GetScaledObject(string name, string @namespace, TriggersPayload triggers,
+            DeploymentV1Apps deployment, int? pollingInterval, int? cooldownPeriod, int? minReplicas, int? maxReplicas)
+        {
+            return new ScaledObjectV1Alpha1
```

Comment:
refactor and reuse common code. Please see https://github.com/tomkerkhove/azure-functions-core-tools/pull/2 for proposed changes.